core/modules/layout/layout.theme.inc

```diff
@@ -277,6 +277,34 @@ function theme_layout_region_inner($variables) {
   return implode('', $variables['blocks']);
 }
 
+
+/**
+ * Output a preview of hero block background image.
+ */
+function theme_layout_hero_background_image($variables) {
+  $element = $variables['element'];
+  $attributes = array();
+  if (isset($element['#id'])) {
+    $attributes['id'] = $element['#id'];
+  }
+  if (!empty($element['#attributes']['class'])) {
+    $attributes['class'] = (array) $element['#attributes']['class'];
+  }
+  $attributes['class'][] = 'form-managed-file';
+
+  $output = '';
+  $output .= '<div' . backdrop_attributes($attributes) . '>';
+  if (isset($element['#file']->uri)) {
+    $output .= '<div class="hero-bg-preview">';
+    $output .= '<img style="height:250px" src="' . image_style_url('medium', $element['#file']->uri) . '" />';
```

Comment:
I think `medium` is a good choice here, but let's also pull the size from the upload element by tacking on `'#image_style' => 'medium'`. That way the image style could be adjusted via form-alter.

This should probably call `theme('image_style')` to render the image.

Any CSS to adjust the image should be in `layout.admin.css`.

Reply:
Calling `theme('image_style')` didn't seem to work with SVG backgrounds, which is why we're using an `img` tag directly. But turns out for SVGs to work,  `theme('image_style')` needs to explicitly set width and height, although the image still renders at the medium dimensions. 

In short, fixed this.